interface/backend/images/models.py

```diff
@@ -11,6 +13,22 @@ class ImageSeries(models.Model):
 
     uri = models.CharField(max_length=512)
 
+    def get_or_create(uri):
+        """
+        Given the absolute uri to a directory with DICOM images of a patient,
+        look up the ImageSeries with the same PatientID and SeriesInstanceUID.
+        If none exists so far, create one.
+        Return a tuple of (ImageSeries, created), where created is a boolean specifying whether the object was created.
+        :return: (ImageSeries, bool)
+        """
+        file_ = glob.glob1(uri, '*.dcm')[0]
+        plan = dicom.read_file(uri + "/" + file_)
```

Comment:
``os.path.join``? Or even Django's ``safe_join`` rather..  Oherwise we could have some fun with ``../../../../lol`` I fear

Reply:
Very good point, thank you @lamby ! I updated it with `safe_join`.